include/FastBVH/BBox.h

```diff
@@ -29,18 +31,31 @@ struct BBox final {
   //! points of the bounding box.
   Vec3 extent;
 
+  //! Gets a bounding box with an
+  //! infinity min and a negative infinity max.
+  static constexpr BBox inf() noexcept {
```

Comment:
This can be sometimes useful in loops where the minimum and maximum of a set of boxes is being calculated.